### PR TITLE
Use dep instead of Glide

### PR DESCRIPTION
**Pull request by dtan4**
## WHY

Golang team develops [dep](https://github.com/golang/dep) for dependency management which will be included in standard tools. 
Glide recommends to migrate to dep.

> Please consider trying to migrate from Glide to dep. 

https://github.com/Masterminds/glide/tree/3e13fd16ed5b0808ba0fb2e4bd98eb325ccde0a1#golang-dep

## WHAT

Migrate dependency management to dep.

### CODE REVIEW

**Comment by dtan4** [comment]
wait for Go 1.9 support... https://github.com/wantedly/developers-account-mapper/pull/42

**Comment by dtan4** [comment]
@koudaiii plz review

File: glide.lock [hunk 1]
```diff
@@ -1,52 +0,0 @@
-hash: 4f7b5b4be406f8e4818a0fc45065eb3e1ff396d245344bb7901263b06ac6df67
-updated: 2016-12-27T18:24:33.605019579+09:00
-imports:
-- name: github.com/armon/go-radix
-  version: 4239b77079c7b5d1243b7b4736304ce8ddb6f0f2
-- name: github.com/aws/aws-sdk-go
-  version: 0df2a7ae7e82d2643e4a8fed1b2e36a1cf0580bc
```
Comment (koudaiii) [inline glide.lock:7]: https://github.com/aws/aws-sdk-go/commit/0df2a7ae7e82d2643e4a8fed1b2e36a1cf0580bc

📝 "Release v1.6.4"

**Comment by koudaiii** [comment]
LGTM 💨 💨

**Pull request merged**